src/main.rs

```diff
@@ -165,19 +185,20 @@ async fn set_eip_status_created(
     let params = PatchParams::default();
     let result = eip_api.patch_status(eip_name, &params, &patch).await;
     if result.is_ok() {
-        info!("Done updating status");
+        event!(Level::INFO, "Done updating status for created EIP.");
```

Comment:
Thankfully in a tracing world, I think these kinds of log statements are kinda redundant.

Reply:
Mostly. They are still nice for the stdout.